### PR TITLE
RED-1218 replace fontawesome witn fontello

### PR DESCRIPTION
**Pull request by janjaapdriessen**

### CODE REVIEW

File: js/angular_xeditable/resources/js/xeditable.js [hunk 1]
```diff
@@ -1,85 +1,85 @@
 /*!
 angular-xeditable - 0.6.0
 Edit-in-place for angular.js
-Build date: 2016-12-27 
+Build date: 2016-12-27
```
Comment (janjaapdriessen) [inline js/angular_xeditable/resources/js/xeditable.js:4]: PR best viewed with `?w=1`, my editor ate the whitespace

**Comment by richardicke** [comment]
This is a public repo; this change will now break the package for people that use vanilla FA.

**Comment by janjaapdriessen** [comment]
I reflected the fontawesome.io -> fontello.com change in the changes.txt

**Review by thefunny42 — CHANGES_REQUESTED** [review]
Minified version need to be updated.

**Comment by janjaapdriessen** [comment]
not using these changes in the end, we override template in our own application.

**Pull request closed without merging**